graphql/types.go

```diff
@@ -1151,13 +1151,14 @@ func (c Connector) MarshalJSON() ([]byte, error) {
 // (See the examples for ways to expand an existing type).
 type ConnectorConfig struct {
 	// ConnectorConfig
-	Connection     RabbitMQConnection `gocti:"connection"      json:"connection,omitempty"`
-	Listen         string             `gocti:"listen"          json:"listen,omitempty"`
-	ListenRouting  string             `gocti:"listen_routing"  json:"listen_routing,omitempty"`
-	ListenExchange string             `gocti:"listen_exchange" json:"listen_exchange,omitempty"`
-	Push           string             `gocti:"push"            json:"push,omitempty"`
-	PushRouting    string             `gocti:"push_routing"    json:"push_routing,omitempty"`
-	PushExchange   string             `gocti:"push_exchange"   json:"push_exchange,omitempty"`
+	Connection        RabbitMQConnection `gocti:"connection"          json:"connection,omitempty"`
+	Listen            string             `gocti:"listen"              json:"listen,omitempty"`
+	ListenRouting     string             `gocti:"listen_routing"      json:"listen_routing,omitempty"`
+	ListenExchange    string             `gocti:"listen_exchange"     json:"listen_exchange,omitempty"`
+	ListenCallbackUri string             `gocti:"listen_callback_uri" json:"listen_callback_uri,omitempty"`
```

Comment:
Do we know for what is this `ListenCallbackUri` used ?

Reply:
Ref: https://github.com/OpenCTI-Platform/opencti/issues/10046
This is quite cool actually. That would have been nice for our setup with the agent machine (we don't have it anymore)